app/utils.py

```diff
@@ -26,19 +26,17 @@ def read_bitswan_yaml(bitswan_dir: str) -> dict[str, Any] | None:
 
 
 async def call_git_command(*command, **kwargs) -> bool:
-    host_dir = os.environ.get(
-        "BITSWAN_GITOPS_DIR_HOST", "/home/root/.config/bitswan/local-gitops/"
-    )
+    cwd = kwargs.get("cwd")
     host_path = os.environ.get("HOST_PATH")
     host_home = os.environ.get("HOST_HOME")
     host_user = os.environ.get("HOST_USER")
 
     # If all host environment variables are set, use nsenter to run git command on host
-    if host_dir and host_path and host_home and host_user:
+    if cwd and host_path and host_home and host_user:
```

Comment:
This is too many  positional arguments to format which leads to readability issues. Can you use a format string instead? 
```
host_command = (
    f'PATH={host_path} su - {host_user} -c "cd {cwd} && PATH={host_path} HOME={host_home} '
    f'{" ".join(shlex.quote(arg) for arg in command)}"'
)
```